src/verus.pest

```diff
@@ -863,7 +877,7 @@ visibility = {
 
 attr_core = {
     trigger_attribute
-    | "#" ~ bang_str? ~ "[" ~ meta ~ "]"
+    | "#" ~ bang_str? ~ "[" ~ !"verusfmt" ~ meta ~ "]"
```

Comment:
This bit here is added so that _if_ we are missing `verusfmt::skip` support somewhere, then it'll fail with a parse error, rather than it attempting to reformat that bit (which would likely be more surprising to a user)